portfolio/src/main/webapp/script.js

```diff
@@ -87,35 +104,30 @@ async function getDataJson(path) {
   // When the request is complete return text from response
   return data;
 }
-async function include(path,extension)
-{
+async function include(path,extension){
     const element = document.getElementById(`include-${path}`);
     if(element)
     {
         const header = await getDataText(`/${path}${extension}`);
         element.innerHTML = header+'\n'+element.innerHTML;
     }
 }
-async function includeHTML(fileName)
-{
+async function includeHTML(fileName){
     include(fileName,".html");
 }
-async function includeDynamicHTML(fileName)
-{
+async function includeDynamicHTML(fileName){
     include(fileName,"");
 }
-function jsListToHtml(list)
-{
+function jsListToHtml(list){
```

Comment:
Please pick a consistent style for function definitions throughout this file. I think the most common one for Javascript is:

```javascript
function jsListToHtml(list) {
  ...
}
```

although any consistent style is a good style.